project/AkkaBuild.scala

```diff
@@ -412,7 +413,7 @@ object AkkaBuild extends Build {
     ) ++
     resolverSettings
 
-  lazy val baseSettings = Defaults.defaultSettings
+  lazy val baseSettings = Defaults.coreDefaultSettings
 
   lazy val parentSettings = baseSettings ++ Seq(
```

Comment:
`baseSettings` can be removed all together as it only contains `Defaults.coreDefaultSettings` which is added by the Core AutoPlugin to all projects.